src/Profile/index.js

```diff
@@ -1,5 +1,14 @@
-let getInfo = () => Promise.resolve()
-let setInfo = () => Promise.resolve()
+import Settings from '../Settings/index'
+import { defaultProfile } from './defaults'
+
+let getInfo = key => {
+  const profile = { ...defaultProfile, ...Settings.get('app', 'profile') }
```

Comment:
I think profile should be a "platform" setting, instead of an "app" setting

Reply:
Changed "app" to "platform"